aten/src/ATen/native/BinaryOps.cpp

```diff
@@ -232,7 +211,7 @@ static inline Tensor& comparison_op_impl_out(Tensor& result, const Tensor& self,
 template <typename Stub>
 Tensor& comparison_op_out(Tensor& result, const Tensor& self, const Tensor& other, Stub& stub) {
   TORCH_CHECK(result.scalar_type() == kBool,
-              "The output tensor of lt must be a bool, but was ", result.scalar_type());
+              "The output tensor of a comparison or logical op must be a bool, but was ", result.scalar_type());
```

Comment:
I'm confused -- the documentation shows the output being uint8, but this error messages says it can be a bool.  What's up?